database/migrations/2019_12_05_164700_alerts_disable_on_update_current_timestamp.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+
+class AlertsDisableOnUpdateCurrentTimestamp extends Migration
+{
+
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        \DB::statement("ALTER TABLE `alerts` CHANGE `timestamp` `timestamp` timestamp NOT NULL default CURRENT_TIMESTAMP;");
```

Comment:
Please use the "real" migration way,
`$this->timestamp('timestamp')->useCurrent();` maybe works.

Reply:
Thank you for the review
I got this issue https://stackoverflow.com/questions/34774628/how-do-i-make-doctrine-support-timestamp-columns
Tried 
```
        Schema::table('alerts', function (Blueprint $table) {
            $table->datetime('timestamp')->useCurrent()->change();
        });
```

Reply:
I see. Annoying.